packages/applicant/src/pages/submissions/[submissionId]/equality-and-diversity/[grantBeneficiaryId]/organisation.page.test.tsx

```diff
@@ -0,0 +1,246 @@
+import { render, screen } from '@testing-library/react';
+import { merge } from 'lodash';
+import { GetServerSidePropsContext, Redirect } from 'next';
+import { parseBody } from 'next/dist/server/api-utils/node';
+import { RouterContext } from 'next/dist/shared/lib/router-context';
+import React from 'react';
+import {
+  getGrantBeneficiary,
+  postGrantBeneficiaryResponse,
+} from '../../../../../services/GrantBeneficiaryService';
+import { createMockRouter } from '../../../../../testUtils/createMockRouter';
+import NextGetServerSidePropsResponse from '../../../../../types/NextGetServerSidePropsResponse';
+import { getJwtFromCookies } from '../../../../../utils/jwt';
+import EqualityAndDiversityPage, {
+  getServerSideProps,
+  OrganisationPageProps,
+  OrganisationRadioOptions
+} from './organisation.page';
+
+jest.mock('next/dist/server/api-utils/node');
+jest.mock('../../../../../services/GrantBeneficiaryService');
+jest.mock('../../../../../utils/jwt');
+
+const renderWithRouter = (ui: React.ReactNode) => {
+  render(
+    <RouterContext.Provider value={createMockRouter({})}>
+      {ui}
+    </RouterContext.Provider>
+  );
+};
+
+describe('Sex page', () => {
+  const customProps = {
+    formAction: '/testFormAction',
+    defaultChecked: null,
+  } as OrganisationPageProps;
+
+  const component = <EqualityAndDiversityPage {...customProps} />;
+
+  describe('UI', () => {
+    it('Renders expected UI', () => {
+      renderWithRouter(component);
+      screen.getByRole('heading', {
+        name: 'Which type of organisation are you applying for a grant on behalf of?',
+      });
+      screen.getByRole('radio', {
+        name: OrganisationRadioOptions.VCSE,
+      });
+      screen.getByRole('radio', { name: OrganisationRadioOptions.SME });
+      screen.getByRole('radio', { name: OrganisationRadioOptions.NEITHER });
+      screen.getByRole('button', { name: 'Continue' });
+      screen.getByRole('button', { name: 'Skip this question' });
+    });
+  });
+
+  describe('GetServerSideProps', () => {
+    const expectedServiceErrorRedirectObject = {
+      redirect: {
+        destination: `/service-error?serviceErrorProps={"errorInformation":"Something went wrong while trying to upload your equality and diversity responses","linkAttributes":{"href":"/submissions/testSubmissionId/equality-and-diversity","linkText":"Please return","linkInformation":" and try again."}}`,
+        statusCode: 302,
+      } as Redirect,
+    };
+
+    const getContext = (overrides: any = {}) =>
+      merge(
+        {
+          params: {
+            submissionId: 'testSubmissionId',
+            grantBeneficiaryId: 'testGrantBeneficiaryId',
+          } as Record<string, string>,
+          query: {
+            returnToSummaryPage: null,
+          } as Record<string, string>,
+          req: {
+            method: 'GET',
+          },
+          resolvedUrl: '/testResolvedURL',
+        } as GetServerSidePropsContext,
+        overrides
+      );
```

Comment:
We can rm the `any` here

```suggestion
    const getContext = (overrides: GetServerSidePropsContext | {} = {}) =>
      merge(
        {
          params: {
            submissionId: 'testSubmissionId',
            grantBeneficiaryId: 'testGrantBeneficiaryId',
          } as Record<string, string>,
          query: {
            returnToSummaryPage: null,
          } as Record<string, string>,
          req: {
            method: 'GET',
          },
          resolvedUrl: '/testResolvedURL',
        } as GetServerSidePropsContext,
        overrides
      ) as GetServerSidePropsContext<EqualityAndDiversityParams, PreviewData>;
```

You'll need to import `EqualityAndDiversityParams` & `PreviewData`